rt_entt_codegen/shared/parallel.cc

```diff
@@ -0,0 +1,233 @@
+#include "parallel.hh"
+
+#include <set>
+
+#include "ecsact/lang-support/lang-cc.hh"
+#include "ecsact/cpp_codegen_plugin_util.hh"
+
+struct loop_iterator_fields {
+	std::vector<ecsact_system_like_id>&                system_list;
+	const std::vector<ecsact_system_like_id>::iterator begin;
+	std::vector<std::vector<ecsact_system_like_id>>&   parallel_system_cluster;
+};
+
+auto loop_iterator(
```

Comment:
keep private functions in this compilation unit
```suggestion
static auto loop_iterator(
```